impl/src/test/java/io/kroxylicious/testing/kafka/KafkaClusterTest.java

```diff
@@ -234,15 +244,13 @@ private void createTopic(AdminClient admin1, String topic, short replicationFact
         admin1.createTopics(List.of(new NewTopic(topic, 1, replicationFactor))).all().get();
     }
 
-    @BeforeEach
-    void before(TestInfo testInfo) throws IOException {
-        this.testInfo = testInfo;
-        this.keytoolCertificateGenerator = new KeytoolCertificateGenerator();
-    }
-
-    @AfterEach
-    void after() {
-        Path filePath = Paths.get(keytoolCertificateGenerator.getCertLocation());
-        filePath.toFile().deleteOnExit();
+    @AfterAll
+    public static void cleanUp() throws IOException {
```

Comment:
for completeness we should probably clear `clusterSizesIndex` here as well.

Reply:
done